src/components/visualizations/chart/highcharts/getOptionalStackingConfiguration.ts

```diff
@@ -292,11 +297,13 @@ export default function getOptionalStackingConfiguration(
     chartOptions: IChartOptions,
     config: any,
     chartConfig: IChartConfig = {},
+    drillConfig?: IDrillConfig,
 ) {
     const { type } = chartOptions;
     return includes(supportedStackingAttributesChartTypes, type)
         ? merge(
               {},
+              setDrillConfigToXAxis(drillConfig),
```

Comment:
Please add simple test for this

Reply:
Do you mean to add test cases to `setDrillConfigToXAxis` itself?
Or add test cases to function calling `setDrillConfigToXAxis`? :)

Reply:
Fixed :)